scripts/run_backend_tests.py

```diff
@@ -659,6 +691,44 @@ def check_coverage(
     process = subprocess.run(
         cmd, capture_output=True, encoding='utf-8', env=env,
         check=False)
+
+    coverage_output_lines = process.stdout.split('\n')
+    # The `process.stdout` is a string containing the coverage report,
+    # structured as follows:
+    # process_stdout_structure =
+    # Name       Stmts Miss Branch BrPart Cover Missing.
+    # -------------------------------------------------.
+    # file1.py      10    0      2      0   100%   12, 14.
+    # file2.py      20    0      4      0    90%   32, 34, 36, 38.
+    # -------------------------------------------------
+    # TOTAL        30    0      6      0   95%.
+    incomplete_coverage_lines: List[str] = []
+    # Lines with incomplete coverage are appenede to incomplete_coverage_lines.
+    for line_number, line in enumerate(coverage_output_lines):
+
+        # The line having incomplete coverage, not a dashed line(-----),
+        # and not the heading line is appended to incomplete_coverage_lines.
+
```

Comment:
Drop this newline, the comment should be directly above the line it is for.

Reply:
@seanlip i have made this change in new commit.